docs/docs/attestation/vuln.md

```diff
@@ -153,7 +153,11 @@ $ trivy image --format cosign-vuln --output vuln.json alpine:3.10
 
 ### Sign with a local key pair
 
-Cosign can generate key pairs and use them for signing and verification. Read more about [how to generate key pairs](https://docs.sigstore.dev/cosign/key-generation).
+Cosign can generate key pairs and use them for signing and verification. After you run the following command, you will get a public and private key pair. Read more about [how to generate key pairs](https://docs.sigstore.dev/cosign/key-generation).
+
+```bash
+$ cosign generate-key-pair
+```
```

Comment:
I've added examples of commands so viewers don't have to look at other sites.  
This is the same modification I made to the SBOM.
https://github.com/aquasecurity/trivy/pull/2690/commits/8cb9cf8f8b2111d5050e38062d86bee6ad131b98